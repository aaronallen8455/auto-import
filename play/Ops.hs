module Ops where

(+++) :: Int -> Int -> Int
k +++ b = k + b

data a ::: b = Mk
  { method :: a -> b
  , (***) :: a -> b -> ()
  }
