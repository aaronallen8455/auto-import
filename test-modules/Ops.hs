module Ops where

(+++) :: Int -> Int -> Int
k +++ b = k + b

class a ::: b where
  a :: a -> b
  (***) :: a -> b -> ()
