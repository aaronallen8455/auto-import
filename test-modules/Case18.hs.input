module Case18 where

import Ops ((+++))

main :: IO ()
main = a ::: a *** a
