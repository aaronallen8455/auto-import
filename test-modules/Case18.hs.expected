module Case18 where

import Ops ((+++), (:::)(a, (***)))

main :: IO ()
main = a ::: a *** a
